### PR TITLE
Implement Map

### LINKED ISSUE #69: Implement Map
A map with borders, size to be defined

Epic: #54

### PR DESCRIPTION
**Pull request by Adan0k**
### Description

Added map to level_1 scene

### Image/Video

image

### Related Issue

**Closes #69**

### Does this PR introduce a breaking change? ⚠️

- [x] No
- [ ] Yes 

### Tests

- [ ] I tested all game E2E;
- [x] I just tested some scenes;
- [ ] Not applicable.

### Check list

Before submitting this pull request, please ensure the following are completed:

- [ ] [My commits follow the project conventions](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#commit-message-convections).
- [ ] My code follows the project's coding style and conventions.
- [ ] I have updated the project documentation (if necessary).
- [ ] My code makes sense with the design.

## Additional Notes
The border already have colision, to the player don't go out of map

---

Thank you for your contribution to Olympus Gone Wild! Your effort is greatly appreciated, and it helps make this project better for everyone. 😄🎮🚀

### CODE REVIEW

**Comment by Gabriel-Sous-a** [comment]
the branch name should be like this [wiki](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#branch-naming-convention)

**Review by Gabriel-Sous-a — CHANGES_REQUESTED** [review]
After testing saw that the map is implemented however there are no borders. And maybe you should expand a bit the map

**Comment by 201flaviosilva** [comment]
> After testing saw that the map is implemented however there are no borders. And maybe you should expand a bit the map

Captura de ecrã 2023-10-13 223308

yap not working

File: main/assets/y5w2xwP.png
Comment (201flaviosilva) [inline main/assets/y5w2xwP.png:1]: Please reference where you got this image and confirm if we can use it, if it is free to be used in the project, we don't want to have legal problems

And give a better name to this file, like TileFlor :)

File: main/level_1.tscn [hunk 1]
```diff
@@ -1,13 +1,314 @@
...
+16:16/0 = 0
+
+[sub_resource type="TileSet" id="TileSet_t1sd6"]
+sources/0 = SubResource("TileSetAtlasSource_l7jy2")
+
 [node name="level1" type="Node2D"]
 process_mode = 1
 script = ExtResource("1_d3jfb")
 
+[node name="TileMap" type="TileMap" parent="."]
```
Comment (201flaviosilva) [inline main/level_1.tscn:307]: Create a scene for this, probably we will use it in more places, and clears this file :)

**Review by 201flaviosilva — CHANGES_REQUESTED** [review]

**Comment by Gabriel-Sous-a** [comment]
When started the player is not centered on the map:

Screenshot 2023-10-16 at 11 25 44

And the borders still dont work:

Screenshot 2023-10-16 at 11 25 19

**Comment by Gabriel-Sous-a** [comment]
In the PR description the tests are applicable, and the commits aren't following the project conventions so please have that checked

File: main/assets/y5w2xwP.png
Comment (Gabriel-Sous-a) [reply main/assets/y5w2xwP.png:1]: TileFloor* 😁

**Comment by Adan0k** [comment]
> When started the player is not centered on the map:
> 
> Screenshot 2023-10-16 at 11 25 44
> 
> And the borders still dont work:
> 
> Screenshot 2023-10-16 at 11 25 19
> 

When I was testing, i placed the player next to the border to not lose time walking.

**Comment by Gabriel-Sous-a** [comment]
Like I said before the branch name is not following the project conventions, neither is the PR. But the borders are working!

**Pull request closed without merging**